src/Document/TextPathElement.ts

```
@@ -0,0 +1,1369 @@
+						pathCmd.start.x,
+						pathCmd.start.y,
+						pathCmd.points[0],
+						pathCmd.points[1],
+						pathCmd.start.x,
+						pathCmd.start.y
+					);
+					break;
+
+				case 'A':
+					const start = pathCmd.points[4];
+					// 4 = theta
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)